components/brave_wallet_ui/components/desktop/views/explore_web3/web3_dapp_details.tsx

```diff
@@ -0,0 +1,210 @@
+      >
+        <ProgressRing />
+      </Column>
+    )
+  }
+
+  // Dapp not found, redirect to the list
+  if (!dapp) {
+    return <Redirect to={WalletRoutes.Web3} />
+  }
+
+  return (
+    <Column
+      fullHeight
+      fullWidth
+      justifyContent='flex-start'
+    >
+      <Column
+        gap={'8px'}
+        fullWidth
+        justifyContent='flex-start'
+        alignItems='center'
+        margin={'32px 0px 0px 0px'}
+      >
+        {isHttpsUrl(dapp.logo) ? (
+          <img
+            src={`chrome://image?${dapp.logo}`}
+            width={72}
+            height={72}
+          />
+        ) : (
+          <PlaceholderImage size={72} />
+        )}
+
+        <Text>{dapp.name}</Text>
+
+        <Row gap={'8px'}>
+          {dapp.categories.map((cat) => (
+            <DappCategoryLabel key={cat}>{cat}</DappCategoryLabel>
+          ))}
+        </Row>
+
+        <Text textSize='14px'>{dapp.description}</Text>
+      </Column>
+
+      <Column
+        fullWidth
+        gap={'8px'}
+        margin='24px 0px'
+      >
+        <Row gap={'8px'}>
+          <DappMetric
+            metricName={getLocale('braveWalletActiveWallets')}
+            meticValue={dapp.uaw}
+            isUSDValue={false}
+          />
+          <DappMetric
+            metricName={getLocale('braveWalletTransactions')}
+            meticValue={dapp.transactions}
+            isUSDValue={false}
+          />
+        </Row>
+        <Row gap={'8px'}>
+          <DappMetric
+            metricName={getLocale('braveWalletMarketDataVolumeColumn')}
+            meticValue={dapp.volume}
+            isUSDValue={true}
+          />
+          <DappMetric
+            metricName={getLocale('braveWalletBalance')}
+            meticValue={dapp.balance}
+            isUSDValue={true}
+          />
+        </Row>
+      </Column>
+
+      <Column
+        fullWidth
+        alignItems={'flex-start'}
+        justifyContent={'flex-start'}
+        gap={'8px'}
+        margin='0px 0px 20px 0px'
+      >
+        <Text textAlign='left'>{getLocale('braveWalletNetworks')}</Text>
+        <Row
+          justifyContent={'flex-start'}
+          alignItems={'flex-start'}
+        >
+          {dapp.chains.map((chainName) => {
+            const net = networks.find((n) =>
+              n.chainName.toLowerCase().startsWith(
+                // The dapps list uses '-' to separate words
+                chainName.replaceAll('-', ' ')
+              )
+            )
+            return net ? (
+              <CreateNetworkIcon
+                key={net.chainId}
+                network={net}
+                size='big'
+                marginRight={8}
+              />
+            ) : null
+          })}
+        </Row>
+      </Column>
+
+      <NavButton
+        minWidth='100%'
+        isExternalLink
+        buttonType='primary'
+        onSubmit={() => {
+          if (isHttpsUrl(dapp.website)) {
+            openTab(dapp.website)
+          }
+        }}
```

Comment:
Should we hide this link if it is not https? could be confusing that clicking the link does nothing